common/unsafe/src/main/java/org/apache/spark/unsafe/Platform.java

```diff
@@ -75,67 +76,131 @@ public static boolean unaligned() {
     return unaligned;
   }
 
+  public static int getInt(MemoryBlock object, long offset) {
```

Comment:
Shouldn't this be a part of the memory block api? Instead of platform? Same for all other methods here.

Reply:
Do you want to move them (i.e. methods with `MemoryBlock` argument)  into `unsafe/memory/MemoryBlock.java`?

Reply:
Moved